Sources/SwiftBuildSupport/PackagePIFBuilder+Helpers.swift

```diff
@@ -140,14 +141,14 @@ extension PackageModel.Package {
         self.manifest.displayName
     }
 
-    var packageBaseBuildSettings: SwiftBuild.PIF.BuildSettings {
-        var settings = SwiftBuild.PIF.BuildSettings()
-        settings.SDKROOT = "auto"
-        settings.SDK_VARIANT = "auto"
+    var packageBaseBuildSettings: ProjectModel.BuildSettings {
+        var settings = BuildSettings()
+        settings[.SDKROOT] = "auto"
+        settings[.SDK_VARIANT] = "auto"
 
```

Comment:
This is basically how we change build settings now (i.e., using a `subscript`). 

Brace for this *exact* same code everywhere...